api/views/stories.py

```diff
@@ -24,31 +25,51 @@ def get_stories():
 def post_stories():
     data = request.get_json()
     if 'story_name' in data:
-        new_story = row_constructor(Story, data)
+        if data['story_name'] is not None:

```

Comment:
same here. since both error messages below are basically the same thing, you can just use the message `The story_name parameter was not provided or is invalid`